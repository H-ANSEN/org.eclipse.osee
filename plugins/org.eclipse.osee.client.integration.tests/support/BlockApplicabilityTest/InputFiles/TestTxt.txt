This is a text file that is meant for testing. 

/* Feature[JHU_CONTROLLER] */
This line should not be seen
/* End Feature */

/* Feature[ROBOT_SPEAKER=SPKR_A] */
This line should be seen
/* End Feature */