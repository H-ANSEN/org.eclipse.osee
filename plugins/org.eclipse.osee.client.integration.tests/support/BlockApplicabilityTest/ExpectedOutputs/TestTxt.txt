This is a text file that is meant for testing. 


This line should be seen
